tests/inventory/hosts

```diff
@@ -1,2 +1,2 @@
-localhost ansible_connection=local ansible_python_interpreter="/usr/bin/env python"
+localhost ansible_connection=local ansible_python_interpreter="python"
```

Comment:
(Why) Is this really needed?

Reply:
The way ansible works and the way the test environment is configured enforces it.
Pipeline adds desired python version in non-standard path. If `ansible_python_interpreter` is not set, ansible uses default python to execute commands:
 debug tasks:
```
     - name: Set up Python
        uses: actions/setup-python@v5
        with:
          python-version: ${{ matrix.python }}
      - name: debug
        run: |
          which python
          echo ${PATH}
          ansible localhost -vvv -i tests/inventory/hosts -m command -a "echo $PATH"
```
part of debug output, `ansible_python_interpeter` not set:
```
/opt/hostedtoolcache/Python/3.12.4/x64/bin/python
/opt/hostedtoolcache/Python/3.12.4/x64/bin:/opt/hostedtoolcache/Python/3.12.4/x64:/snap/bin:/home/runner/.local/bin:/opt/pipx_bin:/home/runner/.cargo/bin:/home/runner/.config/composer/vendor/bin:/usr/local/.ghcup/bin:/home/runner/.dotnet/tools:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin
ansible [core 2.13.13]
  config file = /home/runner/work/squeezer/squeezer/ansible.cfg
  configured module search path = ['/home/runner/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/pipx/venvs/ansible-core/lib/python3.8/site-packages/ansible
  ansible collection location = /home/runner/work/squeezer/squeezer/build/collections
  executable location = /opt/pipx_bin/ansible
  python version = 3.8.10 (default, Nov 22 2023, 10:22:35) [GCC 9.4.0]
  jinja version = 3.1.4
  libyaml = True
Using /home/runner/work/squeezer/squeezer/ansible.cfg as config file
host_list declined parsing /home/runner/work/squeezer/squeezer/tests/inventory/hosts as it did not pass its verify_file() method
script declined parsing /home/runner/work/squeezer/squeezer/tests/inventory/hosts as it did not pass its verify_file() method
auto declined parsing /home/runner/work/squeezer/squeezer/tests/inventory/hosts as it did not pass its verify_file() method
Parsed /home/runner/work/squeezer/squeezer/tests/inventory/hosts inventory source with ini plugin
Skipping callback 'default', as we already have a stdout callback.
Skipping callback 'minimal', as we already have a stdout callback.
Skipping callback 'oneline', as we already have a stdout callback.
META: ran handlers
<localhost> ESTABLISH LOCAL CONNECTION FOR USER: runner
<localhost> EXEC /bin/sh -c 'echo ~runner && sleep 0'
<localhost> EXEC /bin/sh -c '( umask 77 && mkdir -p "` echo /home/runner/.ansible/tmp `"&& mkdir "` echo /home/runner/.ansible/tmp/ansible-tmp-1719834657.3056092-[18](https://github.com/maciej-markowski/squeezer/actions/runs/9743651669/job/26887617068?pr=2#step:5:19)99-220843398698913 `" && echo ansible-tmp-1719834657.3056092-1899-220843398698913="` echo /home/runner/.ansible/tmp/ansible-tmp-1719834657.3056092-1899-220843398698913 `" ) && sleep 0'
<localhost> Attempting python interpreter discovery
<localhost> EXEC /bin/sh -c 'echo PLATFORM; uname; echo FOUND; command -v '"'"'python3.10'"'"'; command -v '"'"'python3.9'"'"'; command -v '"'"'python3.8'"'"'; command -v '"'"'python3.7'"'"'; command -v '"'"'python3.6'"'"'; command -v '"'"'python3.5'"'"'; command -v '"'"'/usr/bin/python3'"'"'; command -v '"'"'/usr/libexec/platform-python'"'"'; command -v '"'"'python2.7'"'"'; command -v '"'"'/usr/bin/python'"'"'; command -v '"'"'python'"'"'; echo ENDFOUND && sleep 0'
<localhost> EXEC /bin/sh -c '/usr/bin/python3.8 && sleep 0'
Using module file /opt/pipx/venvs/ansible-core/lib/python3.8/site-packages/ansible/modules/command.py
```
It uses default python 3.8 instead of 3.12.

Setting interpreter through `/usr/bin/env` does not work anymore as per https://github.com/ansible/ansible/issues/83476. Since the `actions/setup-python@v5` updates the `PATH` variable, setting the interpreter like this `ansible_python_interpreter="python"` makes it work.

Reply:
So you are saying it never worked the way we thought it did?
Well, i guess, fine. ;) And thanks for digging it up.

Reply:
I am saying it was working previously, but now it is not for ansible >=2.17 as per https://github.com/ansible/ansible/issues/83476. 
I noticed it here https://github.com/pulp/squeezer/actions/runs/9665363409/job/26662489592?pr=164, it errors out with ` "module_stderr": "/bin/sh: 1: /usr/bin/env python: not found\n",`